runtime/include/tt/runtime/detail/debug.h

```diff
@@ -63,13 +63,19 @@ struct Hooks {
 #endif
   }
 
+  void unregisterHooks() const {
+#if defined(TT_RUNTIME_DEBUG) && TT_RUNTIME_DEBUG == 1
+    operatorCallback = std::nullopt;
+#endif
+  }
+
```

Comment:
I think did you absorb Collin's PR?

Reply:
yeah, waiting on this merge: https://github.com/tenstorrent/tt-mlir/pull/1334